src/edu/ucsd/library/xdre/tab/TabularEditRecord.java

```diff
@@ -288,6 +288,25 @@ public void overlayElements() throws Exception {
                     }
                 }
             }
+
+            // File: file use  //////////////////////////////////////////////////////////////
+            if (key.equalsIgnoreCase("file use") || key.equalsIgnoreCase("file use 2")) {
```

Comment:
is `file use 2` a convention that's being used for this? I've not seen that before and am curious to learn

Reply:
@mcritchlow It's the headers provided in the Excel Standard InputStream for the second file: `File name 2` and  `File use 2`. And we have `File name` and  `File use` for the master file respectively.

Reply:
Ah, thank you!